content/en/guide/v10/getting-started.md

```diff
@@ -225,27 +223,12 @@ These rewrites are configured using regular expressions in your Jest configurati
 }
 ```
 
-#### Aliasing in Snowpack
-
-To alias in [Snowpack](https://www.snowpack.dev/), you'll need to add a package import alias to the `snowpack.config.mjs` file.
+#### Aliasing in TypeScript
 
-```js
-// snowpack.config.mjs
-export default {
-  alias: {
-    "react": "preact/compat",
-    "react-dom/test-utils": "preact/test-utils",
-    "react-dom": "preact/compat",
-    "react/jsx-runtime": "preact/jsx-runtime",
-  }
-}
-```
+TypeScript, even when used alongside a bundler, has its own process of resolving types.
```

Comment:
I know I've seen a few users questioning why TS needs to be configured in addition to their bundler, hence this text. Hopefully it addresses those questions